### PR TITLE
Selective live alert reward notifications

### PR DESCRIPTION
**Pull request by sichangi**
Initial implementation for live alerts, ready for testing on slack

### CODE REVIEW

**Comment by sichangi** [comment]
Commands:

Start Live watch: `/wf alert-watch:start`

Stop Live watch: `/wf alert-watch:stop`

Add a reward: `/wf alert-watch:add:Nitain extract`

Remove a reward: `/wf alert-watch:remove:Nitain extract`

File: src/slack/messaging.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ function sendRegularMessage(text, channel, attachments = []) {
     headers: { Authorization: `Bearer ${config.slackOauth}` },
     data: {
       text: text,
+      as_user: true,
```
Comment (MarkNjunge) [inline src/slack/messaging.js]: This should be taken as a parameter.

**Review by MarkNjunge — CHANGES_REQUESTED** [review]

**Comment by MarkNjunge** [comment]
I've added a contributing guide, make sure to follow those guidelines.

**Comment by MarkNjunge** [comment]
Make sure to test your changes before making a pr.

**Comment by sichangi** [comment]
Minor tweaks implemented, adherence to lint rules and response updates

**Review by MarkNjunge — APPROVED** [review]

File: src/shared/warframe/live/alerts.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+const {alerts} = require("../worldstate");
+const db = require("../../db");
+const winston = require("winston");
+const cron = require("node-cron");
+const {sendRegularMessage} = require("../../../slack/messaging");
```
Comment (MarkNjunge) [inline src/shared/warframe/live/alerts.js:5]: Code specific to Slack should not be here.

File: src/slack/warframe.js [hunk 3]
```diff
@@ -112,6 +115,11 @@ function help(requestBody) {
       fallback: "cetus",
       text: `Returns the current time on and bounties cetus e.g. \`${command} cetus\``
     },
+    {
+      title: "alerts-watch",
+      fallback: "alerts-watch",
+      text: `Manages your live alert notifications e.g. \`${command} alerts-watch:[start/stop/add/remove]:[reward]\``
```
Comment (MarkNjunge) [inline src/slack/warframe.js:121]: Provide more detailed examples, similar to your [second comment](https://github.com/MarkNjunge/helix/pull/1#issuecomment-415349830) in this PR.

**Review by MarkNjunge — CHANGES_REQUESTED** [review]

**Comment by MarkNjunge** [comment]
I'm closing this PR since there hasn't been any update. Feel free to re-open it or create another one.

**Pull request closed without merging**